extensions/kermit-koin/build.gradle.kts

```diff
@@ -40,17 +41,17 @@ kotlin {
     tvosX64()
 
     linuxX64()
-
     mingwX64()
 
-    // TODO: These targets aren't supported by Koin yet:
+    // TODO: These targets aren't supported by Koin yet or stopped the support:
+    //    watchosArm32()
```

Comment:
With all the other cleanup, perhaps this might be the time to delete the commented out targets?

Reply:
I thought it's there to avoid confusion for reader around why we don't have those targets in `kermit-koin` while being there in `kermit` in general. So I kept it